cypress/integration/brewery/InstanceView.spec.js

```diff
@@ -2,14 +2,40 @@
 // Licensed under the MIT license.
 
 import utils from '../../commons/TestUtils';
+import { setup } from '../../commons/utils/setup';
+import { stub } from '../../commons/services/stubbing';
+import { DEFAULT_WORKSPACE } from '../../fixtures/stubbing/default/workspaces';
 import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
 import { InstanceVisualization, Scenarios, ScenarioManager, Login } from '../../commons/actions';
 
 Cypress.Keyboard.defaults({
   keystrokeDelay: 0,
 });
 
-describe('Instance view', () => {
+const BASE_URL = Cypress.config().baseUrl;
+const url = BASE_URL + '/' + DEFAULT_WORKSPACE.id;
+
+describe('Instance view disabled', () => {
```

Comment:
Sure to add another cypress test instead of a jest of getTabsForCurrentWorkspace ?

Reply:
In my opinion, testing the `getTabsForCurrentWorkspace` function would only test a filter on an array (and maybe the `ConfigUtils.isInstanceViewConfigValid` function if it is not already tested elsewhere).
The cypress test, on the other hand, may catch many more bugs:
- reception of the workspace data by the webapp (especially the `instanceView` object, when `undefined`)
- merging workspace data with the "overrides" config
- make sure that changes when the filter is applied correctly leads to the tab not being shown

Furthermore, the new cypress test can be fully stubbed and does not run simulations, it shouldn't add much time to the existing tests :slightly_smiling_face: